DETAILED_INSTALL.md

```diff
@@ -2,251 +2,79 @@
-
-    * PettingZoo:
-
-        ```
-        make build version=pz
-        ```
-
-    * SMAC: The StarCraft Multi-Agent Challenge Environments :
-
-        Install StarCraft II using a bash script, which is a slightly modified version of the script found [here][pymarl]:
-
-        ```
-        ./bash_scripts/install_sc2.sh
-        ```
-
-        Build Image
-
-        ```
-        make build version=sc2
-        ```
-
-    * Flatland:
-
-        ```
-        make build version=flatland
-        ```
-
-    To allow for agent recordings, where agents evaluations are recorded and these recordings are stored in a `/recordings` folder:
-
-    ```
-    make build version=[] record=true
-    ```
-
-2. Run an example:
+    To then build the docker image run
 
     ```bash
-    make run example=dir/to/example/example.py
+    make build
     ```
 
-    For example, `make run example=examples/petting_zoo/sisl/multiwalker/feedforward/decentralised/run_mad4pg.py`.
-
-    Alternatively, run bash inside a docker container with mava installed, `make bash`, and from there examples can be run as follows: `python dir/to/example/example.py`.
-
-    To run an example with tensorboard viewing enabled, you can run
+2. Run a system:
 
     ```bash
-    make run-tensorboard example=dir/to/example/example.py
+    make run example=dir/to/system.py
     ```
 
-    and navigate to `http://127.0.0.1:6006/`.
-
-    To run an example where agents are recorded (**ensure you built the image with `record=true`**):
-
-    ```
-    make run-record example=dir/to/example/example.py
-    ```
+    For example, `make run example=mava/systems/ff_ippo_rware.py`.
 
-    Where example, is an example with recording available e.g. `examples/debugging/simple_spread/feedforward/decentralised/run_maddpg_record.py`.
+    Alternatively, run bash inside a docker container with mava installed by running `make bash`, and from there systems can be run as follows: `python dir/to/system.py`.
 
 ### Python virtual environment
 
 1. If not using docker, we strongly recommend using a
     [Python virtual environment](https://docs.python.org/3/tutorial/venv.html)
-    to manage your dependencies in order to avoid version conflicts. Please note that since Launchpad only supports Linux based OSes, using a python virtual environment will only work in these cases:
+    to manage your dependencies in order to avoid version conflicts.
 
     ```bash
     python3 -m venv mava
     source mava/bin/activate
     pip install --upgrade pip setuptools
+    git clone https://github.com/instadeepai/Mava.git
+    cd mava
     ```
 
-    1.1  To install the core libraries, including [Reverb](https://github.com/deepmind/reverb) - our storage dataset , Tensorflow and [Launchpad](https://github.com/deepmind/launchpad) - for distributed agent support :
-
-    * Install swig for box2d:
-
-    ```bash
-    sudo apt-get install swig -y
-    ```
+    1.1  To install the core libraries, please run:
```

Comment:
I'd remove this whole section and just put the `pip install .` inside the bigger block

Reply:
Agree, done.